env-files/torch/create_container_env.sh

```diff
@@ -57,10 +58,14 @@ if [ "$1" == "23.09-py3" ]; then
     # Fix needed to compile horovod with torch >= 2.1
     # https://github.com/horovod/horovod/pull/3998
     # Assume that Horovod env vars are already in the current env!
-    pip install --no-cache-dir git+https://github.com/thomas-bouvier/horovod.git@compile-cpp17 || exit 1
+    pip3 install --no-cache-dir git+https://github.com/thomas-bouvier/horovod.git@compile-cpp17 || exit 1
 
     # Install Pov4ML
-    pip install --no-cache-dir "prov4ml[linux]@git+https://github.com/matbun/ProvML@main" || exit 1
+    if [[ "$OSTYPE" =~ ^darwin ]] ; then
+      pip install "prov4ml[apple]@git+https://github.com/matbun/ProvML@main" || exit 1
+    else
+      pip install "prov4ml[linux]@git+https://github.com/matbun/ProvML@main" || exit 1
+    fi
```

Comment:
our containers base Linux images atm, so I would leave only the prov4ml installation with `linux`extras